### ISSUE TITLE
Email integration

### ISSUE THREAD

**Issue by tso** [uwhumansvszombies/uwhvz#14]
Not sure what this looks like. Depends on #17.

**Comment by tiffanynwyeung** [comment]
Need to investigate how to send emails through the website. 

Options:
- https://sendgrid.com/docs/Integrate/Frameworks/django.html offers 100 free emails per day post-free trial. 
- use the current hvz gmail to be the sender email

**Comment by tiffanynwyeung** [comment]
Closed due to the fact that we have a working email integration for mods to contact players so far, but no Wagtail integration to send Wagtail posts (e.g. missions & announcements made on Wagtail) to players and that would probably be a separate issue for v2.0.
